source/includes/APIReference/Users/Examples/_create-response.md.erb

```diff
@@ -24,6 +24,8 @@
         "client_url": "api_sample_output",
         "company_name": "API Sample Output Company",
         "profile_image_url": "",
+        "display_name": null,
```

Comment:
I notice all of the examples have nulls for both new properties.  Any particular reason we're not using example values?

Reply:
Field is null for everyone until the feature is enabled (which could be in a while). The docs on the User Object page describes this.